hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/codegen/JooqCodeGenerator.java

```diff
@@ -55,10 +60,11 @@
   private static final Logger LOG =
       LoggerFactory.getLogger(JooqCodeGenerator.class);
 
-  private static final String SQLITE_DB =
-      System.getProperty("java.io.tmpdir") + "/recon-generated-schema";
-  private static final String JDBC_URL = "jdbc:sqlite:" + SQLITE_DB;
-
+  private static final String DB = Paths.get(
+      System.getProperty("java.io.tmpdir"),
+      "recon-generated-schema-" + Time.monotonicNow()).toString();
+  public static final String RECON_SCHEMA_NAME = "RECON";
```

Comment:
Generally, schemas are lowercase.

Reply:
Yes, but internally, I found that it becomes uppercase in the DB metadata. Hence, if I keep it as "recon", I have to do .toUpperCase() in Jooq code gen setSchemaName for it to find our classes.